src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -1117,6 +1118,10 @@ int RadosStore::read_topics(const std::string& tenant, rgw_pubsub_topics& topics
   return 0;
 }
 
+int RadosStore::stat_topics_v1(const std::string& tenant, optional_yield y, const DoutPrefixProvider *dpp) {
+  return rgw_stat_system_obj(dpp, svc()->sysobj, svc()->zone->get_zone_params().log_pool, topics_oid(tenant), nullptr, nullptr, y, nullptr);
```

Comment:
it may actually be better to send a read request here instead of a stat so we can initialize the metadata cache. if we stat here and go on to read it later, that requires two separate round trips to the osd. what do you think?

Reply:
it is only during migration that we expect this to return sucess.
if we are in v2 after migration we expect -ENOENT, and hopefuly just get a negative chache entry.
if we are in v2, we should not call this at all.

Reply:
sorrry, was a typo. i meant in v1 (where we expect to get actual values) we shhoulod not call this at all

Reply:
np. i just wanted to point out the interaction with stat and metadata caching